### DISCUSSION TITLE
The best flavor?

### DISCUSSION THREAD

**Discussion by kasergus** [catppuccin/catppuccin#2210] [Polls]
I'm not sure about my favorite catppuccin flavor, so I need your help!

**Comment by 7valkio** [comment]
I personally like the latte and mocha flavours.

**Comment by vanillyn** [comment]
this is a tough one, as i love all of them besides macchiato, but id say latte is the prettiest and mocha is the easiest on my eyes. i use frappe

**Comment by stellophiliac** [comment]
i personally like macchiato, frappe is too gray for me and mocha is too dark for my tastes. macchiato has a nice balance

**Comment by PoorPocketsMcNewHold** [comment]
All. There's no "best" ones. That's why there's different ones, because people have different tastes.

**Comment by FPEEU** [comment]
Machiatto is for me the **best** it's perfect baalance.
